ontopy/ontology.py

```diff
@@ -1062,51 +1062,87 @@ def get_entities(  # pylint: disable=too-many-arguments
     def classes(self, imported=False):
         """Returns an generator over all classes.
 
-        If `imported` is `True`, will imported classes are also returned.
+        If `imported` is `True`, classes in imported ontologies
+        are also returned.
+        """
+        return self._entities("classes", imported=imported)
+
+    def _entities(
+        self, entity_type, imported=False
```

Comment:
I think it will be good to document the different values that `entity_type` can take.